function/bool_parameter.go

```diff
@@ -70,6 +71,14 @@ type BoolParameter struct {
 	// alphabetical character and followed by alphanumeric or underscore
 	// characters.
 	Name string
+
+	// Validators is a list of bool validators that should be applied to the
+	// parameter.
+	Validators []BoolValidator
+}
+
```

Comment:
```suggestion
// BoolValidators returns the Validators field value.
```

Or something along those lines for the Go docs.